dataactbroker/fileRoutes.py

```diff
@@ -18,7 +17,7 @@ def add_file_routes(app,CreateCredentials,isLocal,serverPath,bcrypt):
     @permissions_check(permission="writer")
     def submit_files():
         fileManager = FileHandler(request,isLocal=IS_LOCAL, serverPath=SERVER_PATH)
-        return fileManager.submit(LoginSession.getName(session), CreateCredentials)
```

Comment:
Good call. Even without the global changes, this could've been derived as such within the submit method instead of being passed as a parameter, no?